### ISSUE TITLE
Applied decision counts in Source metrics do not match totals from filtering Decisions view

### ISSUE THREAD

**Issue by dhobern** [CatalogueOfLife/checklistbank#1052]
If I look at the decisions in the COL project, and filter for ILDIS, I find a total of 960 decisions:

https://www.checklistbank.org/catalogue/3/decision?limit=100&mode=update&offset=900&subjectDatasetKey=1015 - 958 update decisions

https://www.checklistbank.org/catalogue/3/decision?limit=100&mode=block&offset=0&subjectDatasetKey=1015 - 2 block decisions

In source metrics, the count is given as 958 applied decisions:

https://www.checklistbank.org/catalogue/3/sourcemetrics

If I try the same with World Plants, the number of decisions listed for the dataeset is 34801 (34686 update and 135 block). The Source metrics shows 34643 applied decisions.

In neither case are any decisions shown as broken.

Are the block decisions not counted, or is the  (or not applied)?

**Comment by mdoering** [comment]
You are right that the block decisions have not been counted for. Only update decisions are counted as the idea was to count the number of "changes applied" that can be found in the final data, rather than the filtering decisions. Do you think it would be better to also count them? In the case of block decisions they do block out entire subtrees with potentially many names. We cannot easily count those at least not without a hit in performance in some cases.

**Comment by mdoering** [comment]
We could also have the applied decision count be a map with the decision mode as the key if that is important?
There is also an `ignoredCounter` map in the metrics that tracks the names that have been ignored for fixed rules:

>   CHRESONYM,
>   INDETERMINED,
>   RANK,
>   INCONSISTENT_NAME,
>   // ignored name types
>   NAME_PLACEHOLDER,
>   NAME_NO_NAME,
>   NAME_HYBRID_FORMULA,
>   NAME_INFORMAL;
